server/src/instant/model/schema.clj

```diff
@@ -186,7 +190,7 @@
 
 (defn cascade-message [[etype label]]
   (str etype "->" label ": "
-       "Cascade delete is only possible on one-to-one and one-to-many attributes. "
+       "Cascade delete is only possible on cardinality: one attributes. "
```

Comment:
Could we write this in a way that matches what the user writes in their `instant.schema.ts` file? I'm worried they might be confused since we don't use `cardinality` in that file.

Would it be correct to say "only possible on links with `has: "one"`" or "not possible on links with `has: "many"`"?